### ISSUE TITLE
Accesss to EditProfileAdminForm

### ISSUE THREAD

**Issue by stratisastro** [miguelgrinberg/flasky#432]
Unless I'm missing something I can't see how the EditProfileAdminForm is accessed by the administrator to edit a user's details. I was expecting something like a facility to search for a user by id, name or email which would then open the form for editing. Was this an exercise left for the reader or is there something I'm missing?

I'm looking at the master version (label 17g), having done flask db upgrade.  I've created an admin user (who is recognised as such by Flasky) and a couple of test standard users.

Many thanks

Nigel

**Comment by miguelgrinberg** [comment]
You have to go to the profile page of the user you want to edit. There you have a button to access this form. Here is the button code in the user.html template: https://github.com/miguelgrinberg/flasky/blob/master/app/templates/user.html#L43-L45.

**Comment by stratisastro** [comment]
Many thanks for the quick response Miguel.

I think my problem was that I skipped a few commits so hadn't manually set my admin user to have admin role. When I did that the button appeared.

One minor point:  The route decorator and argument for edit_profile_admin implies it is the id which is being passed, whereas I think it is actually the user name.

Obviously you do also need to know the name to put it in the browser address - I will try and implement a facility for the administrator to browse/search the users by name or email to navigate to their profile.
